.grit/patterns/python/py_no_debugger.md

```diff
@@ -11,14 +11,60 @@ engine marzano(0.1)
 language python
 
 `` as $body where {
```

Comment:
We shouldn't use this pattern at all. Use `import pdb as $db` or `import pdb` as the root then add conditions to that root.

Reply:
Hi @morgante , yes I will try avoid using this anti pattern. ` `` as $body where {}`

updated it now,